pkg/subctl/operator/install/serviceaccount/ensure.go

```diff
@@ -15,6 +15,8 @@ import (
 
 //go:generate go run generators/yamls2go.go
 
+const OperatorServiceAccout = "submariner-operator"
```

Comment:
Typo (no particular need to fix it, don’t update the PR just for this): `Account`.

Reply:
oops, I can do ti quickly, np

Reply:
Done, just use squash when merging.